Unit Tests/Tests/Store/StoreTests.swift

```diff
@@ -0,0 +1,172 @@
+import XCTest
+import PusherPlatform
+@testable import PusherChatkit
+
+class StoreTests: XCTestCase {
```

Comment:
Are we running these tests on the CI at the moment?
There is just one check but it is related to Synk.

Reply:
I think we are not and that is because there is workspace anymore. We can change that now, but that might affect running tests on `master`.